DPNUnet.imjoy.html

```diff
@@ -94,14 +93,11 @@
 from scipy.misc import imread
 import base64
 from shutil import copyfile
-
+from PIL import Image
+from io import BytesIO
```

Comment:
This two should be removed

Reply:
the modules are used in the function "plot_tensors"

Reply:
I see, ok thanks.